### PR TITLE
DOCS-10287-PM_AddCertificateKeystore_ESB

### PR DESCRIPTION
**Pull request by emileemooreVT**
…w feature.

### CODE REVIEW

File: modules/ROOT/pages/create-keystore.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+= Mulesoft Keystore for Endpoints
```
Comment (vijayang) [inline modules/ROOT/pages/create-keystore.adoc]: Please change to "Host organization Keystore for AS2 endpoints"

File: modules/ROOT/pages/create-keystore.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+== Keystore Settings
+
+Using the Mulesoft Keystore, the host xref:Certificates.adoc [certificates] and private key can be stored and reused for endpoint generation.
```
Comment (vijayang) [inline modules/ROOT/pages/create-keystore.adoc]: Please rephrase this as:
You can create a new keystore for the Host organization when configuring AS2 receive and send endpoints, and reuse it in new endpoint configurations by selecting an existing keystore configuration.

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/create-keystore.adoc (hunk 1 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc]: Does "AS2 Host Keystore" work?

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/create-keystore.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
+= AS2 Host Keystore
+
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc:3]: Consider a snippet of transition text here, maybe just to tell what the AS2 host keystore is, when you want or need to interact with it, and what subtopics are included in this topic?

Comment (kbostrom) [reply modules/ROOT/pages/create-keystore.adoc:3]: I sort of wish the title of the topic set were Create an AS2 Host Keystore, to emphasize the user task and goal.

File: modules/ROOT/pages/create-keystore.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+== Keystore Settings
+
+You can create a new keystore for the host when configuring AS2 receive and send endpoints. After creating the keystore, you can reuse it in new endpoint configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: Think about focusing on imperative (active) verbs rather than gerunds:
```suggestion
You can create a new keystore for the host when you configure AS2 receive and send endpoints. After you create the keystore, you can reuse it in new endpoint configurations.
```
Think about this GLOBALLY when writing narrative, especially in a topic set that is or includes a user task.

File: modules/ROOT/pages/create-keystore.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Required
+|Keystore name
+|The unique name for the keystore. The value in this field is automatically generated but can be modified.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: "The unique name for the keystore" raised a question for me about whether the preposition should be "for" or "of." In look this up, I was reminded of the handy Google Ngram Viewer, which provides stats about which term uses are most common: in this case, "name of."

https://books.google.com/ngrams/graph?content=name+of%2Cname+for&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cname%20of%3B%2Cc0%3B.t1%3B%2Cname%20for%3B%2Cc0

Comment (kbostrom) [reply modules/ROOT/pages/create-keystore.adoc]: preposition, passive voice
```suggestion
|The unique name of the keystore. The value in this field is automatically generated, but you can modify it.
```

File: modules/ROOT/pages/create-keystore.adoc [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+|Yes
+
+|<Host> public certificate
+|The digital certificate for your organization. This certificate is based on the RSA algorithm for the AS2 transport protocol.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: certificate of

File: modules/ROOT/pages/create-keystore.adoc [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+|Yes
+
+|<Host> private key file
+|The private key.
+|Yes
+
+|Key passphrase
+|The passphrase for your private key.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc:23]: In this case, the Ngram viewer shows "for" as the most common.

File: modules/ROOT/pages/create-keystore.adoc [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+|Yes
+|===
+
+== Create a Keystore
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc:27]: Consider transition snippet.
Most of our documentation does not proceed from title to first step directly. This is not to say that one is better than the other, just that the transition has been our style. I will raise this with Mysti and see what she thinks, as our library architect.

File: modules/ROOT/pages/create-keystore.adoc [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+. In the left navigation menu, select the host.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: We might start omitting terms like "left" due to accessibility issues. I am wondering how we can do this, if items in the UI are not labeled. Just a heads up. Plus...you and I are working on this per our team V2MOM '22.

File: modules/ROOT/pages/create-keystore.adoc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+. In the *Endpoints* section, click *New*. +
+The *New Endpoint* window appears.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
. In the *Endpoints* section, click *New*. +
+
The *New Endpoint* window appears.
```

Comment (kbostrom) [reply modules/ROOT/pages/create-keystore.adoc]: ...but do we have to describe what the person performing the task can readily see?

File: modules/ROOT/pages/create-keystore.adoc [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+. In the *New Endpoint* window, select values in the *Usage* and *Protocol* fields according to the type of endpoint using the keystore.
+. In the *<host> keystore* section, click *Select*. +
+The *Select a <host> keystore* window appears.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: ditto re. format and necessity

File: modules/ROOT/pages/create-keystore.adoc [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+. Click *New*.
+. In the *New keystore for <host>* window:
+.. In the *Keystore name* field, enter a unique name for the keystore.
+.. In the *<host> public certificate* field, click *Choose file*, navigate to your public certificate, and then click *Open*.
+.. In the *<host> private key file* field, click *Choose file*, navigate to your private key file, and then click *Open*.
+.. In the *Key passphrase* field, enter a passphrase for your private key.
+. Click *Upload keystore*. +
+The newly created keystore appears in the *Select a <host> keystore* window and the keystore is saved. You can either continue to xref:create-endpoint.adoc[create an endpoint] or you can close the window.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
. Click *Upload keystore*. +
+
The newly created keystore appears in the *Select a <host> keystore* window and the keystore is saved. You can either continue to xref:create-endpoint.adoc[create an endpoint] or you can close the window.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 13]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
-| Yes
+|Owner
+|The host or trading partner that owns the endpoint.
+|Yes
 
-| Owner
-| The host or trading partner that owns the endpoint.
-| Yes
+|Usage
+|The usage of the endpoint. For receive endpoints, the value in this field is `Receive from partners`. The value in this field is automatically generated based on the type of endpoint created.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
|How the endpoint is used. For receive endpoints, the value in this field is `Receive from partners`. The value in this field is automatically generated based on the type of endpoint created.
```

Comment (kbostrom) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: We want to avoid "usage" when we can.

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 14]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
+|No
 
-| Protocol
-| The protocol previously selected in the *Protocol* field of the *New endpoint: Receive from partners* window. This value cannot be subsequently modified.
-| Yes
+|Protocol
+|The protocol previously selected in the *Protocol* field of the *New endpoint: Receive from partners* window. This value cannot be subsequently modified.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
|The protocol previously selected in the *Protocol* field of the *New endpoint: Receive from partners* window. You cannot later modify this value.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 15]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
+|Yes
 
-| Description
-| The user-supplied description of the purpose and use of the endpoint.
-| No
+|Endpoint Name
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: See Line 10: Is the UI inconsistent in its capitalization (Keystore name, Endpoint Name)?
(what a surprise....)
We do have to quote the UI verbatim, when quoting. We can fix when writing about.

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 16]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
+|The automatically generated name used to identify the endpoint, which you can subsequently modify manually.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc:23]: Standardize to this wording style for Lines 11 and 19.

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 17]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
-| The private key.
-| Yes
+When disabled, the AS2 receive endpoint application is deployed to public port 8082 by default. The endpoint URL can be directly accessed over the internet.
 
-| Key passphrase
-| The passphrase for your private key.
-| Yes
-|===
+[Note]
+This configuration is applicable only for CloudHub VPC deployments from Partner Manager
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
This configuration is applicable to CloudHub VPC deployments from Partner Manager only.
```

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 18]
```diff
@@ -8,16 +8,24 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
-| Yes
+|Owner
+|The trading partner or host that owns the endpoint.
+|Yes
 
-| Protocol
-| The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
-| Yes
+|Usage
+|The usage of the endpoint. For receive endpoints, the value in this field is `Send to partners`. The value in this field is automatically generated based on the type of endpoint created.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: ditto prev. re. "usage"

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 19]
```diff
@@ -8,16 +8,24 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
+|The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
+|Yes
+
+|Partner
+|The partner to whom the transaction is sent.
+|Yes
 
 |Endpoint Name
-| The name used to identify the endpoint
+|The name used to identify the endpoint
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: In peer elements that use a mixture of fragments and sentences, all elements should end with a period.
When possible, don't mix:
```suggestion
|The name used to identify the endpoint.
```

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 20]
```diff
@@ -26,7 +34,7 @@ The value in the *Endpoint Name* field is generated automatically but can be mod
 |The user-defined field for describing the purpose and use of the endpoint.
 | No
 
-|Trading partner AS2 URL
+|Partner AS2 URL
 |Your trading partner’s AS2 URL to which outbound EDI transmissions need to be sent.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: need to > must

**Review by kbostrom — DISMISSED** [review]
CE, CU

File: modules/ROOT/pages/create-keystore.adoc (hunk 3 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc:3]: changed to create - moved section above settings to the top

File: modules/ROOT/pages/create-keystore.adoc (hunk 6 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc]: done

File: modules/ROOT/pages/create-keystore.adoc (hunk 7 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc:23]: Left alone!

File: modules/ROOT/pages/create-keystore.adoc (hunk 9 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc]: Done

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 15 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: It is lower case - thanks for catching it!

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 16 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-receive.adoc:23]: Done

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 18 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 20 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done

File: modules/ROOT/pages/create-keystore.adoc (hunk 11 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc]: Done

File: modules/ROOT/pages/create-keystore.adoc (hunk 8 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc:27]: Done

File: modules/ROOT/pages/create-keystore.adoc [hunk 21]
```diff
@@ -0,0 +1,48 @@
+= Create AS2 Host Keystore
+You can create a new keystore for the host when you configure AS2 receive and send endpoints. After you create the keystore, you can reuse it in new endpoint configurations.
+
+== Keystore Settings
+Use these to set up AS2 host keystore settings for AS2 receive and send endpoints.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: Use these to set up AS2... or just Use these AS2?
"Use these to set up settings" sounds odd:
```suggestion
Use these AS2 host keystore settings for AS2 receive and send endpoints:
```

File: modules/ROOT/pages/create-keystore.adoc [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+|The private key.
+|Yes
+
+|Key passphrase
+|The passphrase for your private key.
+|Yes
+|===
+
+== Create a Keystore
+You can create and reuse a AS2 host keystore for use in subsequent new endpoint configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: a AS2 > an AS2

File: modules/ROOT/pages/create-keystore.adoc [hunk 23]
```diff
@@ -0,0 +1,48 @@
...
+
+. In the navigation menu, select the host.
+. In the *Endpoints* section, click *New*. +
++
+The *New Endpoint* window appears.
+. In the *New Endpoint* window, select values in the *Usage* and *Protocol* fields according to the type of endpoint using the keystore.
+. In the *<host> keystore* section, click *Select*. +
+. In the *Select a <host> keystore* window, click *New*.
+. In the *New keystore for <host>* window:
```
Comment (kbostrom) [inline modules/ROOT/pages/create-keystore.adoc]: "In...In" sounds odd to me; what do you think about...:
```suggestion
. Complete the fields of the *New keystore for <host>* window:
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 24]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
-| No
+|Description
+|The user-supplied description of the purpose and use of the endpoint.
+|No
 
-| <Host> public certificate
-| The digital certificate for your organization. This certificate is based on the RSA algorithm for the AS2 transport protocol.
-| Yes
+|Private port (when deploying to VPC)
+a|When enabled, the AS2 receive endpoint application is deployed to private port 8092 in the runtime in your xref:runtime-manager::virtual-private-cloud.adoc[CloudHub VPC]. You can manage the access control to the endpoint via the VPC and the Dedicated Load Balancer settings.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: dedicated load balancer (DLB), lowercase GLOBALLY
```suggestion
a|When enabled, the AS2 receive endpoint application is deployed to private port 8092 in the runtime in your xref:runtime-manager::virtual-private-cloud.adoc[CloudHub VPC]. You can manage the access control to the endpoint via the VPC and the dedicated load balancer settings.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 25]
```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
...
 
-| <Host> private key file
-| The private key.
-| Yes
+When disabled, the AS2 receive endpoint application is deployed to public port 8082 by default. The endpoint URL can be directly accessed over the internet.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: intentional passive?

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 26]
```diff
@@ -8,16 +8,24 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
+|Usage
+|How the endpoint is used. For receive endpoints, the value in this field is `Send to partners`. The value in this field is automatically generated based on the type of endpoint created.
+|No
+
+|Protocol
+|The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
+|Yes
+
+|Partner
+|The partner to whom the transaction is sent.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc:24]: Have we consistently treated "partner" as a "who" or a "what"? Standardize throughout.

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/create-keystore.adoc (hunk 22 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/create-keystore.adoc]: Done

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 25 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: Fixed

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 26 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc:24]: It seems to be a "who".

**Review by kbostrom — DISMISSED** [review]
RA

File: modules/ROOT/pages/create-keystore.adoc [hunk 27]
```diff
@@ -0,0 +1,48 @@
...
+|The private key.
+|Yes
+
+|Key passphrase
+|The passphrase for your private key.
+|Yes
+|===
+
+== Create a Keystore
+You can create and reuse an AS2 host keystore for use in subsequent new endpoint configurations.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
To create an AS2 host keystore that you can reuse in subsequent new endpoint configurations:
```

File: modules/ROOT/pages/create-keystore.adoc [hunk 28]
```diff
@@ -0,0 +1,48 @@
...
+
+. In the navigation menu, select the host.
+. In the *Endpoints* section, click *New*. +
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
. In the *Endpoints* section, click *New*. 
```

File: modules/ROOT/pages/create-keystore.adoc [hunk 29]
```diff
@@ -0,0 +1,48 @@
...
++
+The *New Endpoint* window appears.
+. In the *New Endpoint* window, select values in the *Usage* and *Protocol* fields according to the type of endpoint using the keystore.
+. In the *<host> keystore* section, click *Select*. +
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
. In the *<host> keystore* section, click *Select*. 
```

File: modules/ROOT/pages/create-keystore.adoc [hunk 30]
```diff
@@ -0,0 +1,48 @@
...
+. In the *Select a <host> keystore* window, click *New*.
+. Complete the fields of the *New keystore for <host>* window:
+** In the *Keystore name* field, enter a unique name for the keystore.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-keystore.adoc]: These look like substeps so should be ordered.  So .. rather than **

File: modules/ROOT/pages/create-keystore.adoc [hunk 31]
```diff
@@ -0,0 +1,48 @@
...
+** In the *<host> public certificate* field, click *Choose file*, navigate to your public certificate, and then click *Open*.
+** In the *<host> private key file* field, click *Choose file*, navigate to your private key file, and then click *Open*.
+** In the *Key passphrase* field, enter a passphrase for your private key.
+. Click *Upload keystore*. +
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-keystore.adoc]:
```suggestion
. Click *Upload keystore*. 
```

Comment (valkyrie69) [reply modules/ROOT/pages/create-keystore.adoc]: You don't need the + at the end of the line if it's one line. That plus will get rendered in output.

File: modules/ROOT/pages/create-keystore.adoc (hunk 30 above)
Comment (valkyrie69) [reply modules/ROOT/pages/create-keystore.adoc]: Although, I think you intentionally changed them to unordered... ?

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 32]
```diff
@@ -1,45 +1,45 @@
 = AS2 Receive Endpoint Settings
 
-Your organization's AS2 endpoint settings determine where to receive inbound data from trading partners. One AS2 endpoint can receive inbound data from multiple partners.
+Your organization's AS2 endpoint settings determine where to receive inbound data from partners. One AS2 endpoint can receive inbound data from multiple partners.
 
-Create AS2 identifiers for your organization as host and for your trading partners in their respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[trading partner] profile pages.
+Create AS2 identifiers for your organization as host and for your trading partners in their respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[partner] profile pages.
```
Comment (valkyrie69) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
Create AS2 identifiers for your organization as the host and for your trading partners in their respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[partner] profile pages.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 33]
```diff
@@ -1,45 +1,45 @@
...
-| No
+|Description
+|The user-supplied description of the purpose and use of the endpoint.
+|No
 
-| <Host> public certificate
-| The digital certificate for your organization. This certificate is based on the RSA algorithm for the AS2 transport protocol.
-| Yes
+|Private port (when deploying to VPC)
+a|When enabled, the AS2 receive endpoint application is deployed to private port 8092 in the runtime in your xref:runtime-manager::virtual-private-cloud.adoc[CloudHub VPC]. You can manage the access control to the endpoint via the VPC and the dedicated load balancer settings.
```
Comment (valkyrie69) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
a|When enabled, the AS2 receive endpoint application is deployed to the private port 8092 in Mule in your xref:runtime-manager::virtual-private-cloud.adoc[CloudHub VPC]. You can manage the access control to the endpoint via the VPC and the dedicated load balancer settings.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 34]
```diff
@@ -1,45 +1,45 @@
...
 
-| <Host> private key file
-| The private key.
-| Yes
+When disabled, the AS2 receive endpoint application is deployed to public port 8082 by default. You can access the endpoint URL directly over the internet.
```
Comment (valkyrie69) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
When disabled, the AS2 receive endpoint application is deployed to the public port 8082 by default. You can access the endpoint URL directly over the internet.
```

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 35]
```diff
@@ -59,11 +67,11 @@ The value in the *Endpoint Name* field is generated automatically but can be mod
 |Yes
 
 |Filename pattern
-|xref:file-name-pattern.adoc[File name pattern] as required by your trading partner or backend application.
+|xref:file-name-pattern.adoc[File name pattern] as required by your partner or backend application.
 |Yes
 
 |Request receipt
-a|Specify if you want the trading partner to send an MDN receipt back, confirming the receipt of the message.
+a|Specify if you want the partner to send an MDN receipt back, confirming the receipt of the message.
```
Comment (valkyrie69) [inline modules/ROOT/pages/endpoint-as2-send.adoc]:
```suggestion
a|Specify if you want the partner to return an MDN receipt, confirming the receipt of the message.
```

**Review by valkyrie69 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]
